tpm2tools/import.go

```diff
@@ -0,0 +1,88 @@
+package tpm2tools
+
+import (
+	"fmt"
+	"github.com/golang/protobuf/proto"
+	pb "github.com/google/go-tpm-tools/proto"
+	"github.com/google/go-tpm/tpm2"
+	"io"
+)
+
+// Import decrypts the secret contained in an encoded import request.
+// The req parameter should come from server.CreateImportRequest.
+func Import(rw io.ReadWriter, req []byte) ([]byte, error) {
+	request := &pb.ImportRequest{}
+	err := proto.Unmarshal(req, request)
+	if err != nil {
+		return nil, err
+	}
+	ek, err := getEK(*request.Algorithm, rw)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to retrieve EK: %v", err)
+	}
+
+	defer ek.Close()
+
+	session, _, err := tpm2.StartAuthSession(
```

Comment:
Do we want all of these values hard-coded or will different users have different auth setups?